syntax = "proto3";
package portrait;




import "capacity_admin_sdk/model/capacity_admin/task.proto";
import "capacity_admin_sdk/model/capacity_admin/portrait_config.proto";

/*
GetPortraitDetail请求
*/
message GetPortraitDetailRequest {
    /*
    画像 Id
    */
    string portraitId = 1 ;
}

/*
GetPortraitDetail返回
*/
message GetPortraitDetailResponse {
    /*
    任务列表
    */
    repeated capacity_admin.CapacityTask task = 1 ;
    /*
    画像ID
    */
    string instanceId = 2 ;
    /*
    画像名称
    */
    string name = 3 ;
    /*
    画像编码
    */
    string charset = 4 ;
    /*
    画像负责人
    */
    string user = 5 ;
    /*
    画像描述
    */
    string memo = 6 ;
    /*
    画像配置
    */
    capacity_admin.PortraitConfig config = 7 ;
    /*
    任务创建时间
    */
    string ctime = 8 ;
    /*
    任务创建者
    */
    string creator = 9 ;
}

/*
GetPortraitDetailApi返回
*/
message GetPortraitDetailResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    portrait.GetPortraitDetailResponse data = 4 ;
}

